src/components/charts/SummaryBarChart.jsx

```diff
@@ -0,0 +1,139 @@
+
+import {
+  ResponsiveContainer, BarChart, Bar,
+  Tooltip, XAxis, YAxis, LabelList, Cell,
+} from 'recharts';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const BarChartTitle = styled.div`
+  text-align: center;
+  border-bottom: solid black 1px;
+  line-height: 10px;
+  height: 22px;
+`;
+
+const BarChartWrapper = styled.div`
+  padding: 12px;
+  background-color: white;
+  & svg {
+    overflow: visible;
+  }
+  width: 100%;
+  height: 100%;
+`;
+
+const yAxisStyle = {
+  fontSize: '12px',
+  fontWeight: 'regular',
+  lineHeight: '1em',
+  letterSpacing: '.03rem',
+  color: '#606060',
+};
+
+const labelValueStyle = {
+  fontSize: '10px',
+  fontWeight: 600,
+  lineHeight: '1em',
+  letterSpacing: '.02rem',
+  color: '#3283c8',
+};
+
+const calculateBarChartData = (data, showPercentage, percentageFixedPoint) => {
```

Comment:
this is the same function for pie chart - can take out and put into a helper file somewhere to avoid duplicate code